lib/services/api/user_token_service.rb

```diff
@@ -75,7 +75,7 @@ def log_init(mod, name, options)
     end
 
     def validate_userid(userid)
-      raise "Invalid userid #{userid} specified" unless User.exists?(:userid => userid)
+      raise "Invalid userid #{userid} specified" unless User.where('lower(userid) = ?', userid.downcase).exists?
```

Comment:
@jvlcek You should add the region scoping here. I think it was missed wth the original code and is a bug waiting to happen. We should only allow users that are defined in the current region to log in.

Reply:
@gtanzillo Wildo. Thank you.